src/viewer/form-viewer.directive.js

```diff
@@ -66,6 +67,8 @@ angular.module('mwFormViewer').directive('mwFormViewer', function ($rootScope) {
                         }
 
                         ctrl.buttons.submitForm.visible=false;
+                        $rootScope.submitButton = ctrl.buttons.submitForm.visible;
```

Comment:
why are you using $rootScope instead of ctrl ?

Reply:
$rootScope is getting changed by rootScope.